online-store/src/Store/AppState.ts

```diff
@@ -0,0 +1,132 @@
+import { Basket } from '../components/Basket';
+import { Filter } from '../components/filter';
+import { Search } from '../components/search';
+import { Sort } from '../components/sort';

```

Comment:
создайте в папке components index.ts файл, в который экспортируйте все из всех файлов, лежащих в этой папке. Артур знает, что. я имею ввиду
импорт будет выглядет так

```suggestion
import { Basket, Filter, Search, Sort } from '../components
```